tests/wasmfs/wasmfs_opfs.c

```diff
@@ -84,13 +85,6 @@ int main(int argc, char* argv[]) {
   assert(stat_buf.st_size == 100);
   emscripten_console_log("truncated to 100");
 
-  err = ftruncate(fd, 1);
-  assert(err == 0);
-  err = fstat(fd, &stat_buf);
-  assert(err == 0);
-  assert(stat_buf.st_size == 1);
-  emscripten_console_log("truncated to 1");
-
```

Comment:
Why was this part removed?

Reply:
It made the test nicer by not cutting off the message and we were already testing shortening files later. But I realize that the underlying state was different in the later shortening. This will be fixed by the more exhaustive testing.

Reply:
sgtm to leave for that testing later.